druid/Cargo.toml

```diff
@@ -11,45 +11,41 @@ keywords = ["gui", "ui", "toolkit"]
 edition = "2018"
 
 [package.metadata.docs.rs]
+# The "svg" and "image" features have doc clashes that cause undefined output in docs.
+# See https://github.com/rust-lang/cargo/issues/6313 for more information.
+# Once cargo doc becomes smart enough to handle multiple versions of the same crate,
+# the "svg" and "image" features should be enabled for the docs.rs output.
+features = ["im"]
 default-target = "x86_64-pc-windows-msvc"
 
 [features]
 x11 = ["druid-shell/x11"]
 svg = ["usvg"]
 
-[badges]
-travis-ci = { repository = "xi-editor/druid" }
```

Comment:
I suppose this was removed intentionally? I could not find this badge anywhere on crates.io or docs.rs :thinking:

Reply:
Yeah that's a remnant of a time when druid still used Travis CI.